recipe/install-py-xgboost.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
 pushd ${SRC_DIR}/python-package
-  ${PYTHON} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  ${PYTHON} setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
Could you please elaborate on why `pip` can't be used here now?

Reply:
@jakirkham @beckermr The reason is similar to what happened in https://github.com/conda-forge/staged-recipes/pull/11926: `setup.py` pulls from `libxgboost.so` in the `build` directory, and somehow `pip install .` fails.

Reply:
I will open a new issue in XGBoost and see if `pip install .` can be made to work. Right now, it does not.

Reply:
Thanks for the context. I think that is what [this patch]( https://github.com/conda-forge/xgboost-feedstock/blob/696ca06a4f429484a2c970b3c67c1f807481e7a9/recipe/0001-conda-Unbundle-libxgboost.-dll-dylib-so.patch ) is addressing. Does that address the problem sufficiently from your perspective or are there other things we should be doing?

Reply:
@jakirkham The problem actually has to do with the recent refactoring of `setup.py` in https://github.com/dmlc/xgboost/pull/5271, so the patch does not apply. We will need to find a new way to make `pip` work with the new `setup.py`.

Reply:
Ah ok. Should we drop the patch here then?

Reply:
I actually modified the 0001 patch so that it only adds an entry to libpath.